python/tank/platform/engine.py

```diff
@@ -1052,7 +1054,10 @@ def register_command(self, name, callback, properties=None):
         # to highlight this state. This is used by the tank_command
         # execution logic to correctly dispatch the callback during
         # runtime.
-        arg_spec = inspect.getargspec(callback)
+        if six.PY2:
```

Comment:
Silences an annoying warning that showed up 30 times or so in the `pytest` logs.